📃 Docs about how to install extension manually (PR from TinaCMS) (#439)
SSWConsulting/SSW.Products: content/docs/YakShaver/install-extension-manually.mdx

---
title: Manual Installation of Browser Extension
date: '2025-08-26T16:00:00.000Z'
---

If you are unable to access the Chrome Web Store (for example, if you are in China), you can install the extension manually using this guide.

> Please Note: This method is intended for testing and requires Developer Mode. For the best experience, including automatic updates, install from the Chrome Web Store if you can.

Reviewer: Better to link the store link
```suggestion
Note: This method is intended for testing purposes and requires Developer Mode. For the best experience and automatic updates, install it from the [Chrome Web Store](https://chromewebstore.google.com/detail/yakshaver/glcoihkbmomiblcbpnlkafcipcfnikjc)
```
